web-client/src/pages/contact.js

```diff
@@ -1,10 +1,71 @@
 import React from 'react';
 import '../App.css';
+import Informations from '../components/information'
+import GoogleMaps from '../components/maps'
+import picture from '../images/empty-image.png'
 
-function Contact() {
-    return (
+
+function Contact(props) {
+  const inf1 = {
+    path: picture,
+    link: "mailto:conference@gmail.com",
+    name: "conference@gmail.com" 
+  };
+
+  const inf2 = {
+    path: picture, 
+    link: "tel:987654321",
+    name: "987654321" 
+  };
+
+  const inf3 = {
+    path: picture,
+    link: "#",
+    name: "Parkowa 11/12" 
+  };
```

Comment:
if it won't have any link, it can be only text (and without hover)

Reply:
the street is not a link anymore because I added if